pkgs/top-level/all-packages.nix

```diff
@@ -36512,4 +36512,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  aitrack = callPackage ../applications/misc/aitrack { };
```

Comment:
Since it is seems to be a QT app, it might be better to use libsForQt5.callPackage and replace and remove libsForQt5 in the package itself

Reply:
thanks for the feedback. I updated the commit accordingly.